modules/ingest-attachment/build.gradle

```diff
@@ -70,7 +71,15 @@ dependencies {
   api "org.apache.poi:poi-ooxml:${versions.poi}"
   api "org.apache.poi:poi-ooxml-lite:${versions.poi}"
   api "org.apache.poi:poi:${versions.poi}"
-  api "commons-codec:commons-codec:1.15"
+  configurations.testRuntimeClasspath {
```

Comment:
This should not go inside the `dependencies { }` block.

Reply:
Thanks. I originally had it outside and then moved it in so it would be closer to the api call that was using that version. I'll move it back out.